core/services/ocr2/plugins/directrequestocr/reporting.go

```diff
@@ -34,6 +36,44 @@ type functionsReporting struct {
 
 var _ types.ReportingPlugin = &functionsReporting{}
 
+var (
+	promReportingPlugins = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_reporting_plugin_restarts",
+		Help: "Metric to track number of reporting plugin restarts",
+	}, []string{"jobID"})
+
+	promReportingPluginsQuery = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_reporting_plugin_query",
+		Help: "Metric to track number of reporting plugin Query calls",
+	}, []string{"jobID"})
+
+	promReportingPluginsObservation = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_reporting_plugin_observation",
+		Help: "Metric to track number of reporting plugin Observation calls",
+	}, []string{"jobID"})
+
+	promReportingPluginsReport = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_reporting_plugin_report",
+		Help: "Metric to track number of reporting plugin Report calls",
+	}, []string{"jobID"})
+
+	promReportingAcceptReports = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_reporting_plugin_accept",
+		Help: "Metric to track number of accepting reports",
+	}, []string{"jobID"})
+
+	promReportingTransmitReports = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_reporting_plugin_transmit",
+		Help: "Metric to track number of transmiting reports",
+	}, []string{"jobID"})
+
+	promReportingTransmitBatchSize = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "functions_reporting_plugin_transmit_batch_size",
+		Help:    "Metric to track batch size of transmitting reports",
+		Buckets: []float64{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 100, 1000},
```

Comment:
Can we have all buckets from 1 to 20 and only then 100 and 1000?

Reply:
Done